.github/workflows/log-coderabbit-review.yml

```
@@ -19,6 +19,90 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Write review cleaner
+        run: |
+          cat > /tmp/clean_review.py << 'PYEOF'
+import re, os, sys
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)